test/integration_test/specs/mysql-app-to-backup/mysql.yaml

```diff
@@ -0,0 +1,72 @@
+kind: StorageClass
```

Comment:
Don't need to redefine the mysql app, you can use `mysql-1-pvc`

Reply:
Fixed.

Reply:
Remove this file, not being used